kentikapi/internal/httputil/httputil.go

```diff
@@ -106,9 +86,12 @@ func defaultHTTPClient() *http.Client {
 // makeRetryPolicy creates customized retry policy.
 // Its implementation is based on retryablehttp.ErrorPropagatedRetryPolicy.
 // Retry policy function returns true if the request should be retried.
-func makeRetryPolicy(statusCodes []int, methods []string) retryablehttp.CheckRetry {
-	statusCodesSet := makeIntSet(statusCodes)
-	methodsSet := makeStringSet(methods)
+func makeRetryPolicy() retryablehttp.CheckRetry {
+	statusCodesSet := makeIntSet([]int{429, 500, 502, 503, 504})
```

Comment:
I think that we can remove status 500 from defaults. community_sdk_python HTTP library doesn't return on 500 and Go gRPC part of code does not return on such "unknown" server errors.

Reply:
Done